pkgs/development/python-modules/pytouchlinesl/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  aiohttp,
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+  pydantic,
+  pytest-asyncio,
+  pytestCheckHook,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pytouchlinesl";
+  version = "0.1.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "jnsgruk";
+    repo = "pytouchlinesl";
+    rev = "refs/tags/${version}";
+    hash = "sha256-xyAy5QtNox1ZeXGQEYXWiEIQKSNQSnRTqr0kgQRmdcg=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    aiohttp
+    pydantic
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-asyncio
+  ];
+
+  pythonImportsCheck = [ "pytouchlinesl" ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
I don't think this is needed for Python packages.

Reply:
Indeed you're right! I missed that.